tensilelite/Tensile/Source/lib/include/Tensile/GranularitySelectionLibrary.hpp

```diff
@@ -188,8 +178,8 @@ namespace TensileLite
                     std::cout << row.second->description() << ": ";
                 }
 
-                if(softwarePredicate(searchType, *(row.second), problem)
-                   && (*row.second->hardwarePredicate)(hardware))
+                if(softwarePredicate(searchType, *(row.second), problem) 
```

Comment:
softwarePredicate with searchType is new from Tensile perspective, to avoid breaking I've added my change in addition to this. @jichangjichang I would appreciate if you could advise on this section about whether it is sufficient.